### PR TITLE
Correct Azure Key Vault Keys .md files content inconsistency and unclear

### PR DESCRIPTION
**Pull request by FredGao-new**
Create PR to address this [issue](https://github.com/Azure/azure-sdk-for-java/issues/10055).

### CODE REVIEW

File: sdk/keyvault/azure-security-keyvault-keys/README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Here is [Azure Cloud Shell](https://shell.azure.com/bash) snippet below to
     }
     ```
 
-* Use the above returned credentials information to set **AZURE_CLIENT_ID**(appId), **AZURE_CLIENT_SECRET**(password) and **AZURE_TENANT_ID**(tenant) environment variables. The following example shows a way to do this in Bash:
+* Use the above returned credentials information to set **AZURE_CLIENT_ID** (appId), **AZURE_CLIENT_SECRET** (password) and **AZURE_TENANT_ID** (tenantId) environment variables. The following example shows a way to do this in Bash:
```
Comment (jongio) [inline sdk/keyvault/azure-security-keyvault-keys/README.md:62]: Oxford comma

**Comment by joshfree** [comment]
@vcolin7 PTAL

File: sdk/keyvault/azure-security-keyvault-keys/README.md [hunk 2]
```diff
@@ -32,12 +32,12 @@ Maven dependency for Azure Key Client library. Add it to your project's pom file
     ```
 
 ### Authenticate the client
-In order to interact with the Key Vault service, you'll need to create an instance of the [KeyClient](#create-key-client) class. You would need a **vault url** and **client secret credentials (client id, client key, tenant id)** to instantiate a client object using the default `AzureCredential` examples shown in this document.
+In order to interact with the Key Vault service, you'll need to create an instance of the [KeyClient](#create-key-client) class. You would need a **vault url** and **client secret credentials (client id, client secret, tenant id)** to instantiate a client object using the default `AzureCredential` examples shown in this document.
```
Comment (vcolin7) [inline sdk/keyvault/azure-security-keyvault-keys/README.md:35]:
```suggestion
In order to interact with the Key Vault service, you'll need to create an instance of the [KeyClient](#create-key-client) class. You would need a **vault url** and **client secret credentials (client id, client secret, tenant id)** to instantiate a client object using the default `DefaultAzureCredential` examples shown in this document.
```

File: sdk/keyvault/azure-security-keyvault-keys/README.md [hunk 3]
```diff
@@ -59,307 +56,309 @@ Here is [Azure Cloud Shell](https://shell.azure.com/bash) snippet below to
...
+Delete an existing key by calling `beginDeleteKey`.
 
 ```java
-keyAsyncClient.beginDeleteKey("keyName")
+keyAsyncClient.beginDeleteKey("<key-name>")
     .subscribe(pollResponse -> {
-        System.out.println("Delete Status: " + pollResponse.getStatus().toString());
-        System.out.println("Delete Key Name: " + pollResponse.getValue().getName());
-        System.out.println("Key Delete Date: " + pollResponse.getValue().getDeletedOn().toString());
+        System.out.printf("Deletetion status: %s\n", pollResponse.getStatus().toString());
```
Comment (g2vinay) [inline sdk/keyvault/azure-security-keyvault-keys/README.md:322]: typo, Deletion

**Pull request merged**